frontend/components/GameSpoilersList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+// eslint-disable-next-line no-unused-vars
+import {AppStore} from '../storage/common';
+// eslint-disable-next-line no-unused-vars
+import { GameSpoilers } from '../storage/models/GameSpoilers';
+interface GameSpoilersListProps{
+	gameSpoilers: Map<number, GameSpoilers>;
+}
+const GameSpoilersList: React.FC<GameSpoilersListProps> = (props: GameSpoilersListProps) =>{
+//I couldn't do destructuring, because i get following error: 'Property 'gameSpoilers' does not exist on type 'Map<number, GameSpoilers>''
+	const gameSpoilers = props.gameSpoilers;
+
+	return (
+		<React.Fragment>
```

Comment:
You can use `<>` instead of `<React.Fragment>`. I personally find the empty one more readable (immediate visual feedback, without even reading, that there is no real outer element).\

Also I might be wrong, but I think you can just return an array of elements, without even wrapping it with anything. So: `return [...gameSpoilers.values()].map(...); ` should work. Give it a try!

Reply:
I gave it a try and I get a type mismatch, if I try to do it, because i return JSXElement instead of React.FC